src/modules/Builder/components/Builder/styles.scss

```diff
@@ -3,3 +3,6 @@
     padding-left: 5px;
   }
 }
+.sidebar-mini.sidebar-collapse .brand-text {
```

Comment:
Let's define these styles in Sidebar component. Now it is simple .jsx file in common/containers, let's create Sidebar dir in common/containers and move the Sidebar files there.